packages/gateway/plugins/auth.ts

```
@@ -53,10 +53,12 @@
   }
 }
 
-const resolveUserFn: ResolveUserFn<UserType, AccounterContext> = async context => {
+export function resolveUser(context: any): UserType | null {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)